src/patternfly/components/AppLauncher/examples/application-launcher.md

```diff
@@ -207,7 +207,7 @@ import './AppLauncher.css'
 
 ### Favorites
 ```hbs
-{{#> app-launcher id="app-launcher-favorites" app-launcher--IsExpanded="true" app-launcher--IsGrouped="true"}}
+{{#> app-launcher app-launcher--IsExpanded="true" app-launcher--IsGrouped="true"}}
```

Comment:
Why do we need to remove the `id`? We reference that in examples, usually to build unique `id` attribute values in the HTML. You can see that a couple of lines down in the `{{#> form-control...` block, it's used in the `concat` function